### PR TITLE
Setup linting

### PR DESCRIPTION
**Pull request by nunocosta-casafari**
# Description

Setup linting.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    */setup.py
+    */wsgi.py
+    */settings.py
+    */forms/*
+    */forms/*
+
+    # POC of logging, remove it when POC is implemented or removed.
+    src/logging/*
+    # POC of monitoring, remove it when POC is implemented or removed.
+    src/monitoring/*
```
Comment (dmytrostriletskyi) [inline setup.cfg]: Remove this, please.

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+per-file-ignores=
+    */test_*.py: D205
+exclude=
+    */shared/*
+    */migrations/*
+    */main.py
+    
+    # POC of logging, remove it when POC is implemented or removed.
+    */logging/*,
+    */monitoring/*
```
Comment (dmytrostriletskyi) [inline setup.cfg]: Remove this, please.

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+    src/monitoring/*
+    src/main.py
+    src/before_request.py
+parallel = True
+
+[coverage:paths]
+; https://github.com/casafari/documentation/blob/master/ops/coveralls/README.md#sending-reports-when-using-docker
+source =
+    ./
+    /types-detection-service/
```
Comment (dmytrostriletskyi) [inline setup.cfg]: Remove this, please.

File: setup.cfg (hunk 1 above)
Comment (nunocosta-casafari) [reply setup.cfg]: done

File: setup.cfg (hunk 3 above)
Comment (nunocosta-casafari) [reply setup.cfg]: done

File: setup.cfg (hunk 2 above)
Comment (nunocosta-casafari) [reply setup.cfg]: done

**Pull request merged**